packages/ember-toucan-core/src/components/form/controls/character-count.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="type-xs-tight" ...attributes>{{this.currentCount}}
+  /
+  {{@max}}</span>
```

Comment:
I'm wondering if there are any RTL concerns about doing this?  Is there someone we could ask for verification?

Reply:
Checked with design, who checked with translation - turns out there is no issue at the moment.